### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Ashuaks**
- In this project i set up a new repository and prepare it with the necessary linters.
- Like instructed, I chose the the first template and build it first page.
- In this project i created a branch called: Mobile-version-Skeleton.
- there is an header which carriers nav in which the images are found.
- In also add the Logo in a link as well as other images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+
+        <nav class="tool_bar">
+            <a href="#" class="my-logo">Egbe Bless Ashuakie</a>
```
Comment (muneebulrehman) [inline index.html:14]: Only the hamburger button should be in the `nav` tag. Consider moving the logo outside it.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+body {
+  margin: auto;
+  width: 375px;
+  background-color: #e5e5e5;
+}
```
Comment (muneebulrehman) [inline style.css:5]: - Optional
Consider removing the fixed width from the body. What it is doing is making the body follow fixed width of 375px, so when the display size increases your app width remains same and it is causing issues as soon as display size increases 375px.
Here you can see the issue with the display size of the iPhone XR.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Ashuaks , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Good PR description

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add a proper Readme file for the project. Follow [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template for a professional readme file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Ashuaks,

## APPROVED 🟢

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**